### DISCUSSION TITLE
Getting started / Build tools

### DISCUSSION THREAD

**Discussion by fabswt** [tailwindlabs/tailwindcss#1682] [General]
Hi there!

I'm finally getting started with TailwindCSS (ordered TailwindUI when it came out, but still have to dive into it!) Here are a few things that are sort of getting in the way…

1. This is not a bug report or a feature proposal, so is this actually the **right place to post**?
  - The template for new issues here says to post "anything else" on https://github.com/tailwindcss/discuss/issues …but it's archived (read-only) 😜.

2. I'm totally **new** to **build tools / asset pipeline tools**. Anything to recommend? The install page lists:

- _Webpack_
- _Gulp_
- _Webpack Encore_
- _Brunch_

(And also _Laravel Mix_ and _Ember.js_, but I guess they're not relevant if you don't use either framework.)

This is one of those questions that's probably primarily opinion-based… or dependent on what you _have_ to use (say at work.) But advice would be welcome; what are you guys using when working with Tailwind? Or what are people using in general with Tailwind?

(I'm using PHP with Slim 4, a micro-framework. So any of them seems like an option.)

Thanks for the help!

**Comment by adamwathan** [comment]
Hey great question! We actually archived the `discuss` repo because we now have access to the new Discussions feature on GitHub, so moving this there!

**Comment by ajitbohra** [comment]
> (And also Laravel Mix and Ember.js, but I guess they're not relevant if you don't use either framework.)

`Laravel Mix` i have seen many people think it can be only used with laravel but that's not true. It's framework agnostic and can be used outside laravel and works well.

`tailwindcss + laravel-mix` gives a quick heard start and you can actually focus on building things.

Have been using this setup for 
- Simple HTML pages
- Laravel App
- React App
- WordPress themes & plugins

And it fits perfectly into the workflow.
